devnet/query-server.yaml

```diff
@@ -55,6 +57,10 @@ spec:
             - containerPort: 6069
               name: rest
               protocol: TCP
+            - containerPort: 6068
+              name: status
+              protocol: TCP
           readinessProbe:
-            tcpSocket:
-              port: rest
+            httpGet:
+              port: 6068
```

Comment:
Do you want to call this "status" instead of "6068" in case you change the port number later on?